File: chain-of-responsibility/java/Image.java
package ChainOfResponsibility;

// Uygulama süresince resimlerimizi temsil edecek sınıf.
class Image {
    public String name;
    public String extension;

    public Image(String name, String extension) {
        this.name = name;
        this.extension = extension;
    }
}

File: chain-of-responsibility/php/Image.php
<?php

// Uygulama süresince resimlerimizi temsil edecek sınıf.
class Image
{
    public string $name;
    public string $extension;

    public function __construct(string $name, string $extension)
    {
        $this->name = $name;
        $this->extension = $extension;
    }

}
